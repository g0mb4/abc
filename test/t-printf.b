main() {
    extrn printf;
    auto a 1, b 2;

    printf("Hello*n");
    printf("d (9) = %d*n", 9);
    printf("o (9) = %o*n", 9);
    printf("c (a) = %c*n", 'a');
    printf("s (Hello) = %s*n", "Hello");
    printf("%% = %%*n");

    printf("%d + %d = %d*n", a, b, a+b);
}
