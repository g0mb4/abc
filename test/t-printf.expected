Hello
d (9) = 9
o (9) = 11
c (a) = a
s (Hello) = Hello
%% = %%
1 + 2 = 3
